src/middleware.js

```diff
@@ -36,5 +36,13 @@ export function middleware(request, event) {
 }
 
 export const config = {
-  matcher: '/writing/:path/'
-}
+  matcher: [
+    {
+      source: "/writing/:path/",
+      missing: [
+        { type: "header", key: "next-router-prefetch" },
+        { type: "header", key: "purpose", value: "prefetch" },
+      ],
+    },
+  ],
+};
```

Comment:
I've been searching for this solution. Thanks a bunch! 😄

Just a small request: Could you use `'` instead of `"`?

```suggestion
  matcher: [
    {
      source: '/writing/:path/',
      missing: [
        { type: 'header', key: 'next-router-prefetch' },
        { type: 'header', key: 'purpose', value: 'prefetch' },
      ],
    },
  ],
};
```